css/main.css

```
@@ -115,3 +122,87 @@ a {
 .cv-emoji {
   margin: 4px;
 }
+
+@media screen and (max-width: 1024px) {
+  .page{
+    width: 100%;
+  }
+  .content {
+    flex-direction: column;
+    margin-top: 55px;
+  }
+  .content .photo {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 50%;
```

Having radius (only for mobile) breaks consistency. I think we should remove the rounded style and make things rectangular for both desktop and mobile.

---

Understood. How about a curved corner instead of a pointed corner? like;
```suggestion
    border-radius: 8px;
```